### PR TITLE
Fix imu tf frame

### PR DESCRIPTION
**Pull request by rafal-gorecki**
### Description

In simulation the tf `imu_link` have diffternt transformation than in real robot.

### Modifications

- use current imu transform for simulation
- merge imu_pos_x imu_pos_y imu_pos_z imu_rot_r imu_rot_p imu_rot_y into `imu_xyz` and `imu_rpy`
- simplify URDF `ns` property

## Summary by CodeRabbit

- **New Features**
  - Introduced a launch argument to specify the Panther robot's hardware version for enhanced configurability.
  - Added a version specification for the Panther software in the robot spawn process.

- **Improvements**
  - Simplified the launch configuration by consolidating IMU-related parameters, streamlining the setup process.
  - Enhanced clarity in the configuration by grouping IMU position and orientation parameters into two consolidated properties.
  - Improved namespace handling for better clarity and reduced potential errors.

### CODE REVIEW

File: panther_description/urdf/gazebo.urdf.xacro [hunk 1]
```diff
@@ -6,12 +6,7 @@
 
     <!-- Read property -->
     <xacro:property name="battery_config" value="${xacro.load_yaml(config_file)}" />
-    <xacro:if value="${namespace == ''}">
-      <xacro:property name="ns" value="" />
-    </xacro:if>
-    <xacro:unless value="${namespace == ''}">
-        <xacro:property name="ns" value="${namespace}/" />
-    </xacro:unless>
+    <xacro:property name="ns" value='${namespace + "/" if namespace else ""}' />
```
Comment (delihus) [inline panther_description/urdf/gazebo.urdf.xacro]: @coderabbitai could you find in our code more patterns like this and make different pull request to apply these changes?

File: panther_gazebo/launch/spawn_robot.launch.py [hunk 2]
```diff
@@ -82,6 +82,7 @@ def generate_launch_description():
         launch_arguments={
             "add_wheel_joints": add_wheel_joints,
             "namespace": namespace,
+            "panther_version": "1.23",
```
Comment (rafal-gorecki) [inline panther_gazebo/launch/spawn_robot.launch.py]:
```suggestion
```

File: panther_description/urdf/gazebo.urdf.xacro (hunk 1 above)
Comment (delihus) [reply panther_description/urdf/gazebo.urdf.xacro]: Yes, please. Create pull request for that

Comment (delihus) [reply panther_description/urdf/gazebo.urdf.xacro]: Could you do this on your own?

**Review by delihus — APPROVED** [review]

**Pull request merged**